sonic_platform_base/sonic_xcvr/sfp_optoe_base.py

```diff
@@ -128,6 +128,15 @@ def set_power_override(self, power_override, power_set):
     def get_eeprom_path(self):
         raise NotImplementedError
 
+    def set_optoe_write_max(self, write_max):
```

Comment:
We should define an abstract `set_write_max(self, write_max)` method in SfpBase and then use this implementation for SfpOptoeBase. Platform vendors that don't use optoe would then implement set_write_max.

Reply:
Sfpoptoe driver keeping the default write max to '1' is done for historical reasons. I don't see any reason why non-optoe drivers should keep the default write max other than SMBUS block write size of 32 bytes. Just wanted to avoid yet another non-implemented API in SfpBase. Do you see any platform that needs this change? if not, we can revisit later.